_posts/2021-02-11-Creating-A-Game.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: Creating A Game
+description: 
+summary: 
+tags:
+---
+Today, I did typing club for about an hour. We also built a game, we had to create a new folder locally and work through a tutorial that Kajari provided for us we finished with a game. [This is the article we used](https://developer.mozilla.org/en-US/docs/Games/Tutorials/2D_Breakout_game_pure_JavaScript). After we were done with the game, we had to push up the game up to a repository in Github, for this we used [This article](https://docs.github.com/en/github/importing-your-projects-to-github/adding-an-existing-project-to-github-using-the-command-line). To end this we had to turn the game into a Github page and add the link onto our Github account.
```

Comment:
` We also built a game, we had to create a new folder locally and work through a tutorial that Kajari provided for us we finished with a game. ` --> This sentence doesn't mane a lot of sense.
Get rid of this part: `for us we finished with a game`.

Start a new paragraph for `After we were done with the game`.

`Github, for this we used` -> Close out the sentence and start a new one: `Github. For this we used`

`[This article]` -> `[this article]` since it's in the middle of the sentence, don't capitalize the `this`.